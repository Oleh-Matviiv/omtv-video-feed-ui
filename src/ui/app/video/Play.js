import React from 'react';
import styled from 'styled-components';

const Play = styled.div.attrs({
  className: 'video-play',
})`
  grid-area: "play";
`;

export default Play;
