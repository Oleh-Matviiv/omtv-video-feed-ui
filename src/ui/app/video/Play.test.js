import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';

import Play from './Play';

describe('snapshots', () => {
  it('Play', () => {
    const wrapper = mount(
      <Play>children</Play>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
