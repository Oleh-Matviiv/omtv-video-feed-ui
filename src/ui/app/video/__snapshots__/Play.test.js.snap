// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots Play 1`] = `
.c0 {
  grid-area: "play";
}

<styled.div>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [
          Object {
            "className": "video-play",
          },
        ],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "c0",
          "rules": Array [
            "
  grid-area: \\"play\\";
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <div
      className="video-play c0"
    >
      children
    </div>
  </StyledComponent>
</styled.div>
`;
